packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -111,13 +111,20 @@ export const CloudPulseRegionSelect = React.memo(
       regions, // Function to call on change
     ]);
 
-    const linodeRegionIds = useFetchOptions({
+    const {
+      values,
```

Comment:
appropriately name the variables
```suggestion
      linodeRegions,
```